### PR TITLE
update to latest browser-style-dictionary

### PR DESCRIPTION
**Pull request by jorenbroekema**
wait for https://github.com/jorenbroekema/style-dictionary/pull/6/

### CODE REVIEW

File: src/node/file-tree-utils.js [hunk 1]
```diff
@@ -279,6 +275,27 @@ export async function repopulateFileTree() {
...
+  fileTreeEl.inputFiles = inputFiles;
+
+  mightDispatchTokens();
+}
+
+let oldTokens = [];
+function mightDispatchTokens() {
+  // Naive compare of the tokens
+  if (
+    JSON.stringify(oldTokens) !== JSON.stringify(styleDictionaryInstance.tokens)
```
Comment (gqio) [inline src/node/file-tree-utils.js:288]: why tokens and not properties ?

Comment (gqio) [reply src/node/file-tree-utils.js:288]: not sure I used, the properties because it's the merged version of all properties, but I may be mistaken

Comment (jorenbroekema) [reply src/node/file-tree-utils.js:288]: I thought that's what the tokens object was 😆

Comment (jorenbroekema) [reply src/node/file-tree-utils.js:288]: I asked Danny Banks:
```
there is not, originally it was called properties, but then the community shifted to using the term design tokens so in 3.0 we added 'tokens', but wanted to be backwards compatible so they both mean the same thing
```

**Pull request merged**